.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,218 @@
+      run: |
+        git config --system core.autocrlf false
+        git config --system core.eol lf
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+        bundler-cache: true
+    - name: Install ruby hrx module
+      run: gem install hrx
+    - name: Install ruby minitest module
+      run: gem install minitest
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Clone and checkout sassc repository
+      run: git clone https://github.com/sass/sassc.git
+    - name: Clone and checkout sass-spec repository
+      run: git clone https://github.com/sass/sass-spec.git
+    - name: Compile libsass ${{ matrix.config.build }} build for ${{ matrix.config.platform }}
+      run: msbuild /m:4 /p:"Configuration=${{ matrix.config.build }};Platform=${{ matrix.config.platform }}" sassc\win\sassc.sln
+    - name: Execute spec test runner
+      run: ruby sass-spec/sass-spec.rb --probe-todo --impl libsass -c sassc/bin/sassc.exe -s sass-spec/spec
+
+  windows-mingw:
+    runs-on: windows-latest
+    name: Windows MinGW build
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {build: shared, platform: x64}
+          - {build: static, platform: x64}
+          - {build: shared, platform: x86}
+          - {build: static, platform: x86}
+
+    steps:
+    - name: Change git config to preserve line-endings
+      run: |
+        git config --system core.autocrlf false
+        git config --system core.eol lf
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+        bundler-cache: true
+    - name: Set up MinGW
+      uses: egor-tensin/setup-mingw@v2
+      with:
+        platform: ${{ matrix.config.platform }}
+    - name: Install ruby hrx module
+      run: gem install hrx
+    - name: Install ruby minitest module
+      run: gem install minitest
+    - name: Clone and checkout sassc repository
+      run: git clone https://github.com/sass/sassc.git
+    - name: Clone and checkout sass-spec repository
+      run: git clone https://github.com/sass/sass-spec.git
+    - name: Add libsass library path to be found
+      if: matrix.config.build == 'shared'
+      run: echo "/d/a/libsass/libsass/lib" >> $GITHUB_PATH
+    - name: Compile libsass ${{ matrix.config.build }} build for ${{ matrix.config.platform }}
+      run: make ${{ matrix.config.build }} BUILD=${{ matrix.config.build }}
+    - name: Copy library over to pass call test
+      if: matrix.config.build == 'shared'
+      run: copy /a/libsass/libsass/lib/libsass.dll sassc/bin/
+    - name: Compile sassc ${{ matrix.config.build }} build for ${{ matrix.config.platform }}
+      run: make sassc BUILD=${{ matrix.config.build }}
+    - name: Execute spec test runner
+      run: ruby sass-spec/sass-spec.rb --probe-todo --impl libsass -c sassc/bin/sassc.exe -s sass-spec/spec
```

Comment:
why using the old ruby runner rather than the new typescript one ?

Reply:
Was enough work to convert the existing code, so this didn't change.
Of course, if somebody wants to convert this, PRs are always welcome!